src/traces/bar/cross_trace_calc.js

```diff
@@ -126,6 +126,8 @@ function initBase(gd, pa, sa, calcTraces) {
         // time. But included here for completeness.
         var scalendar = trace.orientation === 'h' ? trace.xcalendar : trace.ycalendar;
 
+        // TODO skip for multicategory size axis?
```

Comment:
Related to https://github.com/plotly/plotly.js/issues/2391#issuecomment-444545411

Reply:
skipped in https://github.com/plotly/plotly.js/pull/3300/commits/3eac3384e7a40a3873d74957ecb105f1ae60a156